.github/workflows/test.yml

```diff
@@ -18,10 +18,13 @@ jobs:
     steps:
 
     - uses: actions/checkout@v2
-    - name: Use Python ${{ matrix.node-version }}
+    - name: Use Python ${{ matrix.python-version }}
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
     - run: pip install -r requirements-dev.txt
-    - run: pre-commit run --all-files
-    - run: pytest
+    - run: pytest --cov=./notion_client --cov-report=xml
```

Comment:
Let's configure pytest in setup.cfg so that it's not needed to write this! And why the `--cov-report=xml` rather than `term-missing`?

Reply:
no specific reason. I saw xml to be used in example given by codecov. I think codecov supports only XML. Not sure.

As per the docs of the pytest-cov plugin,

```text
--cov-report=TYPE     Type of report to generate: term, term-missing, annotate, html, xml (multi-allowed). term,
                        term-missing may be followed by ":skip-covered". annotate, html and xml may be followed by
                        ":DEST" where DEST specifies the output location. Use --cov-report= to not generate any
                        output.
```

multiple types are allowed. So what about, using both XML and term-missing.

Reply:
> Let's configure pytest in setup.cfg so that it's not needed to write this!

that's a great idea

Reply:
but here is a warning given by pytest

![image](https://user-images.githubusercontent.com/66209958/118357301-16ac1e80-b597-11eb-8198-8d66d59f98da.png)

https://docs.pytest.org/en/6.2.x/customize.html#setup-cfg

Reply:
The use case is simple enough, let's avoid a pyproject.toml for now if we can. We'll move everything in a pyproject.toml when we'll switch to Poetry or another PEP 517 manager.

Reply:
> multiple types are allowed. So what about, using both XML and term-missing.

Let's try without XML first, and if Codecov really needs it, put it back. :)